build/runTest.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+SLUG=$(grep -Pho 'cache/[A-Za-z-\.]+' $1/composer.json | xargs | awk '{print $1}')
+printf "\n\n************ Running tests for $SLUG ************\n\n"
+
+cd $1
+if [ $TRAVIS_PHP_VERSION = 5.5 ]; then composer require --no-update phpunit/phpunit:~4.0; fi
+composer install --no-interaction --prefer-source --ignore-platform-reqs --quiet
```

Comment:
Why do we need ` --ignore-platform-reqs`?
Without it it would install phpunit 4 on php5.5.